electron/libelectron/src/backends/memory_mgmt.cc

```diff
@@ -0,0 +1,110 @@
+#include "memory_mgmt.hh"
+namespace memory_mgmt{
+
+FirstFitAllocator::FirstFitAllocator(size_t dev_mem_size) {
+
+    /* Initialize structures*/
+    device_memory = mmap(NULL,dev_mem_size,PROT_EXEC|PROT_READ|PROT_WRITE,MAP_ANONYMOUS|MAP_SHARED,0,0);
```

Comment:
spaces after commas:
```cpp
device_memory = mmap(NULL, dev_mem_size, PROT_READ | PROT_WRITE, MAP_ANONYMOUS | MAP_SHARED, 0, 0);
```

Also no PROT_EXEC

Reply:
Also still would like to move this outside of the allocator overall

Reply:
Done. Moved to PhotonBackend